qurator/sbb_textline_detector/main.py

```diff
@@ -38,7 +38,7 @@
     """
 
 
-class textlineerkenner:
+class textline_detector:
```

Comment:
PEP8 convention is to use CamelCase for class names.

Reply:
We have not decided for/against PEP8 yet. All other QURATOR modules already use snake_case though.